brokernode-api/controllers/Brokers.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var utils = require('../utils/writer.js');
+var Brokers = require('../service/BrokersService');
+
+module.exports.informBroker = function informBroker (req, res, next) {
```

Comment:
It's easy to know what informBrokerGenesisHash does, but what does informBroker do?  I.e. what is it informing the broker of?

Reply:
This is generated code from swagger.. we will use Edmund's broker API